### PR TITLE
[FX-2109] Artist Series QA

### PR DESCRIPTION
**Pull request by anandaroop**
The type of this PR is: **Bugfix** (← I'm guessing, since I don't see a better fit for a Q/A ticket like this)

This PR resolves **https://artsyproduct.atlassian.net/browse/FX-2109**

### Description

This is a grab bag of visual Q/A fixes for the Artist Series view

### Test Plan

n/a, style changes only

### Screenshots

#trivial

### CODE REVIEW

File: Artsy/View_Controllers/Admin/ARAdminSettingsViewController.m [hunk 1]
```diff
@@ -139,7 +139,7 @@ - (ARCellData *)generateFeaturePage
 - (ARCellData *)generateArtistSeries
 {
     return [self tappableCellDataWithTitle:@"→ Artist Series" selection:^{
-        AREigenArtistSeriesComponentViewController *viewController = [[AREigenArtistSeriesComponentViewController alloc] initWithArtistSeriesID:@"alex-katz-departure"];
```
Comment (anandaroop) [inline Artsy/View_Controllers/Admin/ARAdminSettingsViewController.m:142]: Once non-style change I made in this PR was to point us to an artist series instance that actually has a description text populated.

File: src/lib/Scenes/ArtistSeries/ArtistSeriesMeta.tsx [hunk 2]
```diff
@@ -61,8 +61,8 @@ export const ArtistSeriesMeta: React.SFC<ArtistSeriesMetaProps> = ({ artistSerie
           onPress={() => {
             SwitchBoard.presentNavigationViewController(metaRef.current!, `/artist/${artist.slug}`)
           }}
-          style={{ marginVertical: 10 }}
```
Comment (anandaroop) [inline src/lib/Scenes/ArtistSeries/ArtistSeriesMeta.tsx:64]: Switched from this to using `Spacers` so that the dimensions could be governed by our design system scale.

**Comment by ashleyjelks** [comment]
> Bugfix (← I'm guessing, since I don't see a better fit for a Q/A ticket like this)

Yeah, I've been confused a few times about what to put here as well, although I do love the PR template! I can see how it could be a bugfix or an enhancement!

File: Artsy/View_Controllers/Admin/ARAdminSettingsViewController.m (hunk 1 above)
Comment (ashleyjelks) [reply Artsy/View_Controllers/Admin/ARAdminSettingsViewController.m:142]: Make sense! And as soon as https://github.com/artsy/eigen/pull/3694 is merged we should have access to Artist Series pages via the Artist page as well, so should make for easier QAing!

File: src/lib/Scenes/ArtistSeries/ArtistSeriesMeta.tsx [hunk 3]
```diff
@@ -78,11 +75,12 @@ export const ArtistSeriesMeta: React.SFC<ArtistSeriesMetaProps> = ({ artistSerie
               </TouchableWithoutFeedback>
             }
           />
+          <Spacer my={0.5} />
         </TouchableOpacity>
       )}
       <ReadMore
         data-test-id="description"
-        textStyle="sans"
+        textStyle="new"
```
Comment (ashleyjelks) [inline src/lib/Scenes/ArtistSeries/ArtistSeriesMeta.tsx:83]: I'll have to look into this change, I've not yet seen `textStyle=new`  on the ReadMoor component!

Comment (anandaroop) [reply src/lib/Scenes/ArtistSeries/ArtistSeriesMeta.tsx:83]: Yes, [hot off the press](https://github.com/artsy/eigen/blame/61706a9597b0ddbdccf3c4a388dbe11b7da45ff4/src/lib/Components/ReadMore.tsx#L36-L42)!

File: src/lib/Scenes/ArtistSeries/ArtistSeriesMoreSeries.tsx [hunk 4]
```diff
@@ -72,7 +72,13 @@ export const ArtistSeriesMoreSeries: React.FC<ArtistSeriesMoreSeriesProps> = ({
...
         const artistSeriesItem = item?.node
         if (!!artistSeriesItem) {
-          return <ArtistSeriesMoreSeriesItem artistSeriesItem={artistSeriesItem} handleNavigation={handleNavigation} />
+          return (
+            <ArtistSeriesMoreSeriesItem
+              key={artistSeriesItem.internalID}
+              artistSeriesItem={artistSeriesItem}
+              handleNavigation={handleNavigation}
+            />
+          )
```
Comment (anandaroop) [inline src/lib/Scenes/ArtistSeries/ArtistSeriesMoreSeries.tsx:81]: I've rebased already to undo this change, since it was addressed via #3690

**Comment by anandaroop** [comment]
Thanks, merging before more conflicts show up!
